src/common-principles.md

```diff
@@ -536,12 +554,38 @@ like in the example below.
             "F": {
                 "Description": "Female",
                 "TermURL": "https://www.ncbi.nlm.nih.gov/mesh/68005260"
-            },
+            }
         }
     }
 }
 ```
 
+### Compressed tabular files
+
+Large tabular information, such as physiological recordings, MUST be stored with
+[compressed tab-delineated (TSVGZ) files](glossary.md#tsvgz-extensions) when
```

Comment:
I ack that TSVGZ is already in our spec. But it is odd to me to not be TSV.GZ . FWIW it was 517e2ffb2851a342930a0a866c1813aabdeb1dbf which was a part of the huge monumental effort of 
- https://github.com/bids-standard/bids-specification/pull/1107
but I wonder may be @effigies you remember why you did not go for  `tsv.gz` ?

Reply:
Not sure what you mean, can you suggest how this would be addressed in your opinion?

Reply:
```suggestion
[compressed tab-delineated (TSV.GZ) files](glossary.md#tsvgz-extensions) when
```
and
- https://github.com/bids-standard/bids-specification/pull/1755